test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -3238,67 +3216,15 @@ BOOST_AUTO_TEST_CASE(events_with_same_name)
 			event Deposit(address _addr);
 			event Deposit(address _addr, uint _amount);
 			function deposit() returns (uint) {
-				Deposit();
```

Comment:
now duplicate

Reply:
Where is this duplicated?

Reply:
`events_with_same_name_inherited` and `events_with_same_name_inherited_emit` are duplicated.

Reply:
See below.